Apps/Sandcastle/gallery/Time Dynamic Point Cloud.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Time Dynamic Point Cloud">
+    <meta name="cesium-sandcastle-labels" content="Showcases, 3D Tiles">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shouldAnimate : true
+});
+
+var dates = [
+    '2018-07-19T15:18:00Z',
+    '2018-07-19T15:18:00.5Z',
+    '2018-07-19T15:18:01Z',
+    '2018-07-19T15:18:01.5Z',
+    '2018-07-19T15:18:02Z',
+    '2018-07-19T15:18:02.5Z'
+];
+
+var uris = [
+    '../../SampleData/Cesium3DTiles/PointCloud/PointCloudTimeDynamic/0.pnts',
+    '../../SampleData/Cesium3DTiles/PointCloud/PointCloudTimeDynamic/1.pnts',
+    '../../SampleData/Cesium3DTiles/PointCloud/PointCloudTimeDynamic/2.pnts',
+    '../../SampleData/Cesium3DTiles/PointCloud/PointCloudTimeDynamic/3.pnts',
+    '../../SampleData/Cesium3DTiles/PointCloud/PointCloudTimeDynamic/4.pnts'
+];
+
+function dataCallback(interval, index) {
+    return {
+        uri: uris[index]
+    };
+}
+
+var timeIntervalCollection = Cesium.TimeIntervalCollection.fromIso8601DateArray({
+    iso8601Dates: dates,
+    dataCallback: dataCallback
+});
+
+var pointCloud = new Cesium.TimeDynamicPointCloud({
+    intervals : timeIntervalCollection,
+    clock : viewer.clock,
+    style : new Cesium.Cesium3DTileStyle({
+        pointSize : 5
+    })
+});
+viewer.scene.primitives.add(pointCloud);
+
+var start = Cesium.JulianDate.fromIso8601(dates[0]);
+var stop = Cesium.JulianDate.fromIso8601(dates[dates.length - 1]);
+
+viewer.timeline.zoomTo(start, stop);
+
+var clock = viewer.clock;
+clock.startTime = start;
+clock.currentTime = start;
+clock.stopTime = stop;
+clock.clockRange = Cesium.ClockRange.LOOP_STOP;
+
+viewer.camera.setView({
```

Comment:
Could something like `viewer.zoomTo(pointCloud, ...` work here even if it just uses the first / current point cloud?

Reply:
Yeah this is a good idea. Will bump when ready.